kindtest.sh

```diff
@@ -235,11 +242,18 @@ export JAVA_HOME="${JAVA_HOME:-`type -p java|xargs readlink -f|xargs dirname|xar
 echo 'Clean up result root...'
 rm -rf "${RESULT_ROOT:?}/*"
 
-echo 'Run tests...'
+echo 'Determine test filter ...'
+if [ "${maven_profile_name}" = "wls-image-cert" ]; then
+  test_filter="${test_filter_wls}"
+elif [ "${maven_profile_name}" = "fmw-image-cert" ]; then
+  test_filter="${test_filter_fmw}"
+fi
```

Comment:
if (maven_profile_name)  is not defined  (the current behaviour) what path it should follows ?  It should default to current behavior  where it does not take any maven profile.

Reply:
default is test_filter="**/It*" in line 72